### ISSUE TITLE
No longer contains iPhone SE (1st Generation) frames

### ISSUE THREAD

**Issue by simonnickel** [fastlane/frameit-frames#19]
Version 1607001240 removed some older device frames like the old SE. 

When running frameit, it still tries to match it and finds the new SE. The offsets do not fit though.

I think the device naming is not very useful, the files should match the device names used by Xcode.

**Comment by ArturRuZ** [comment]
Good day!
I Also have a problem with SE. The offsets do not fit though.

**Comment by haemi** [comment]
did you find a solution to this?

**Comment by ArturRuZ** [comment]
Yes, we redrew the frame and replaced it, but I didn't remember details
